From 5bea5b9d8c492aab4aa234a2891bdeca404f6f12 Mon Sep 17 00:00:00 2001
From: Vishak A Kamath <vishak@creoit.com>
Date: Fri, 14 Dec 2018 14:23:42 +0530
Subject: [PATCH 2/3] newBranch

---
 test1.txt | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 test1.txt

diff --git a/test1.txt b/test1.txt
new file mode 100644
index 0000000..4f5450a
--- /dev/null
+++ b/test1.txt
@@ -0,0 +1 @@
+New branch
-- 
2.17.2 (Apple Git-113)

